apps/web/src/components/providers/CommunityAuthProvider.tsx

```diff
@@ -1,23 +1,278 @@
 import { createContext } from 'react';
-import { useCreateAuthCommunity } from '../../auth/useCreateAuthCommunity';
+import { IOrganizationEntity, IUserEntity } from '@novu/shared';
+import { setUser as sentrySetUser, configureScope as sentryConfigureScope } from '@sentry/react';
+import { useQuery, useQueryClient } from '@tanstack/react-query';
+import { HttpStatusCode } from 'axios';
+import { useLDClient } from 'launchdarkly-react-client-sdk';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { ROUTES, PUBLIC_ROUTES_PREFIXES } from '../../constants/routes';
+import { useSegment } from './SegmentProvider';
+import { clearEnvironmentId } from './EnvironmentProvider';
+import { getToken } from '../../auth/getToken';
+import { getTokenClaims } from '../../auth/getTokenClaims';
+import { getUser } from '../../api/user';
+import { switchOrganization as apiSwitchOrganization, getOrganizations } from '../../api/organization';
+import { type AuthContextValue } from './AuthProvider';
 
-export const CommunityAuthContext = createContext<ReturnType<typeof useCreateAuthCommunity>>({
+// TODO: Add a novu prefix to the local storage key
+export const LOCAL_STORAGE_AUTH_TOKEN_KEY = 'auth_token';
+
+const noop = () => {};
+const asyncNoop = async () => {};
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Good idea‏